char-sheet/src/components/NotesTable.tsx

```diff
@@ -30,7 +37,18 @@ function NotesTable({service, editable=EditMode.Live}: {
                   className='textarea textarea-bordered leading-tight w-full scrollbar scrollbar-neutral p-2'
                   placeholder='Enter notes here'
                   value={note}
-                  onChange={ evt => service.set_index(i, evt.target.value) }
+                  onChange={ evt => {
+                    service.set_index(i, evt.target.value)
+                    adjustHeight(evt.target)
+                  }}
+                  wrap='soft'
+                  style={{
```

Comment:
The rest of the styling is done with tailwind, so try stick to tailwind properties.

Instead of this, you probably just want to remove `scrollbar scrollbar-neutral` from the `className` and add `overflow-hidden`

Reply:
I'm referring to the inline style elements